demo/conf.py

```diff
@@ -26,6 +26,8 @@
     'sphinxcontrib.plantuml',
     'sphinx_needs',
     'sphinx.ext.autodoc',
+    'sphinx.ext.imgmath',
```

Comment:
This PR seems to contain 3 updates:

1. warning filter
2. headline CSS classes
3. math equation 

This line seems to be for 3), but the PR contains no rst-files where math equations are used.
Is this missing?

Reply:
This was a bugfix for the missing math equastions from #8. The math equations were and are still contained in the demo.rst file of the demo project. The extension to render them was just missing.